strongbox-npm-metadata-jackson/.gitignore

```diff
@@ -0,0 +1,46 @@
+# common ignore
```

Comment:
There's actually no need for this one. The one in the root of the project is enough.